app/controllers/companies_controller.rb

```diff
@@ -0,0 +1,12 @@
+class CompaniesController < ApplicationController
+  # GET '', subdomain: /.+/
+  def index
+    @company = current_company
```

Comment:
assign an instance variable `@current_company` in application_controller once company is found via request.subdomain and use wherever needed.

`@current_company.issues`
`@current_company.projects`

Reply:
current_company is a helper method

Reply:
yes, it is, but an instance variable `@current_company` once set in `application_controller.rb` it can be used throughout the application in all controllers staying in the same instance,

Reply:
ok